packages/composites/crypto-volatility-index/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`execute with isAdaptive false should return success 1`] = `
+Object {
+  "data": Object {
+    "result": NaN,
+  },
+  "debug": undefined,
+  "jobRunID": "1",
+  "result": NaN,
```

Comment:
I spent quite a bit of time trying to configure the mock data to give valid responses but haven't had any luck.  I tried running the adapter end to end but this didn't work either as the API responses from `https://www.deribit.com/api/v2/public` didn't return a response that allowed the adapter to return a 200.  Open to other suggestions here.

Reply:
@justinkaseman and I noticed an issue when we ran the adapters e2e locally so it looks like even mocking actual responses won't work.